hudi-client/src/test/java/org/apache/hudi/index/bloom/TestHoodieGlobalBloomIndex.java

```diff
@@ -267,12 +267,20 @@ public void testTagLocation() throws Exception {
     for (HoodieRecord record : taggedRecordRDD.collect()) {
       if (record.getRecordKey().equals("000")) {
         assertTrue(record.getCurrentLocation().getFileId().equals(FSUtils.getFileId(filename0)));
+        System.out.println("Record data " + record.getData().toString() + " rowChage 1 " + rowChange1.toString());
+        // assertTrue(record.getData().toString().equalsIgnoreCase(rowChange1.toString()));
```

Comment:
Can you tell me how to compare two HoodieRecords?

Reply:
Use `assertEquals(hoodieRecords1, hoodieRecords2)` should make sense.

Reply:
This does not work. But have fixed it.